lib/stdlib/src/proc_lib.erl

```diff
@@ -522,6 +524,30 @@ translate_initial_call(DictOrPid) ->
 	    {?MODULE,init_p,5}
     end.
 
+%% -----------------------------------------------------
+%% [get] set_process_description/1
+%% Add and fetch process id's to aid in debugging
```

Comment:
We can't use Id because a process identifier is a pid which is something else. 
And *process_description* was suggested in the OTB (which you where at) a year or so ago.

Reply:
I think 11 is not super long, but sure sometimes i make it desc in variable names but that is mostly because I have other parts in front and then it becomes long for sure. But as it is an API name think it is good that it is descriptive instead of short. I think we in many context have referred to it as label, so maybe we should have an new OTB about if label would be better than description. It could have the benefit of  it being less suggestive of being very long string.

Reply:
http://wiki.otp.ericsson.se/wiki/Technical_Board_Decisions#30.2F9-2021_Teams_meeting
That OTB was 2 yrs ago and I knew very little back than ;-).

The note from OTP just presents a naming example and not an exact requirement on how the API should look. Therefore I think it is more important what you (and we) think today after implementing it and knowing more.